scripts/test-ci.sh

```diff
@@ -7,7 +7,7 @@ set -euo pipefail
 # variable is set in dockerfile
 if [ "${SONARCLOUD_ANALYSIS:-}" == "true" ]; then
   echo 'Running tests with coverage and reporter'
-  npm run test -- --coverage --testResultsProcessor jest-sonar-reporter
+  npm run test -- --maxWorkers=50% --coverage --testResultsProcessor jest-sonar-reporter
```

Comment:
Currently the pipeline is failing on cirrus, since Februrary 24th - https://cirrus-ci.com/github/SonarSource/eslint-plugin-sonarjs

This allows it to actually finish and not exceed the allowed 8gb